### PR TITLE
Add Markdownlint action

### PR DESCRIPTION
**Pull request by amandahla**
This PR adds Markdownlint action to check only line length.
In the future and if needed, is possible to add new rules as documented [here](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -282,13 +282,21 @@ provide examples for how to use them.
 
 ## Random Values
 
-While creating tests, sometimes you need to assign values to variables or parameters in order to simulate a user behavior, for example. In this case, instead of using constants or fixed values, consider using random ones generated by [`secrets.token_hex()`](https://docs.python.org/3/library/secrets.html#secrets.token_hex).
+While creating tests, sometimes you need to assign values to variables
+or parameters in order to simulate a user behavior, for example. In this case,
+instead of using constants or fixed values, consider using random ones generated
+by [`secrets.token_hex()`](https://docs.python.org/3/library/secrets.html#secrets.token_hex).
```
Comment (jdkandersson) [inline CONTRIBUTING.md:288]: This line looks like it is longer than 80 characters, primarily due to the link. I was guessing that markdownlint would ask that the link be on its own line?

**Review by jdkandersson — COMMENTED** [review]
Nice, just a small question

File: CONTRIBUTING.md (hunk 1 above)
Comment (amandahla) [reply CONTRIBUTING.md:288]: From [here](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md013---line-length):
"This rule has an exception where there is no whitespace beyond the configured line length. This allows you to still include items such as long URLs without being forced to break them in the middle."

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by jdkandersson — APPROVED** [review]

**Pull request merged**